src/App.tsx

```diff
@@ -2,15 +2,23 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  function handleNewMovie(movie: Movie) {
+    setMovies([...movies, movie]);
```

Comment:
```suggestion
    setMovies(prevMovies => [...prevMovies, movie]);
```

What about this comment?